action.yml

```diff
@@ -47,10 +50,8 @@ runs:
     - name: Install lychee
       run: |
         # Cleanup artifacts from previous run in case it crashed
-        rm -rf "lychee-${{ inputs.LYCHEEVERSION }}-x86_64-unknown-linux-gnu.tar.gz" lychee
-        curl -sLO "https://github.com/lycheeverse/lychee/releases/download/${{ inputs.LYCHEEVERSION }}/lychee-${{ inputs.LYCHEEVERSION }}-x86_64-unknown-linux-gnu.tar.gz"
-        tar -xvzf "lychee-${{ inputs.LYCHEEVERSION }}-x86_64-unknown-linux-gnu.tar.gz"
-        rm "lychee-${{ inputs.LYCHEEVERSION }}-x86_64-unknown-linux-gnu.tar.gz"
+        rm -f lychee
+        curl -sL "https://github.com/lycheeverse/lychee/releases/download/${{ inputs.LYCHEEVERSION }}/lychee-x86_64-unknown-linux-gnu.tar.gz" | tar xvz -
         install -t "$HOME/.local/bin" -D lychee 
         rm lychee
```

Comment:
```suggestion
rm -f lychee
```

Reply:
Technically it's not needed because the archive contains a single binary named `lychee`, but I pushed a future proof version `rm -rf lychee*` which will remove all files and folders which start with `lychee`.